src/main/java/baza/trainee/service/impl/EventServiceImpl.java

```diff
@@ -33,10 +40,19 @@ public Event getById(String id) {
     }
 
     @Override
-    public Event save(EventPublication newEvent) {
-        Event event = eventMapper.toEvent(newEvent);
+    public Event save(EventPublication newEvent, String sessionId) {
+        Event eventToSave = eventMapper.toEvent(newEvent);
+        Event savedEvent = eventRepository.save(eventToSave);
```

Comment:
Why do we save the Event twice?

Reply:
eventToSave - creating an event using  eventMapper.toEvent(newEvent);
delete - Event savedEvent = eventRepository.save(eventToSave);
in return - change savedEvent on eventToSave

Reply:
eventToSave - creating an event using eventMapper.toEvent(newEvent);
delete - Event savedEvent = eventRepository.save(eventToSave);
in return - change savedEvent on eventToSave